GOVERNANCE.md

```diff
@@ -0,0 +1,477 @@
+  point in time, the commit that `master`/`main` points to must successfully
+  build and pass all tests successfully.
+
+  Reaching that goal is possible because of the employed merge strategies (read
+  below).
+- **Merge strategies**
+  Merging is the way how the project accepts and implements changes.
+
+  Because of the pursued "everygreen master", merging is implemented in a
+  certain way that forbids several bad practices which could lead to a breaking
+  build on the `master`/`main` branch, and a special workflow is implemented to
+  ensure not only the "evergreen master" but also to streamline the workflow for
+  integrating pull requests.
+
+  The following actions are **not** allowed:
+
+  * Committing to `master`/`main` directly
+  * Squash-Merging of pull requests (which is equivalent to committing to
+    `master`/`main` directly).
+    The github repository is configured to not allow squash merges.
+  * Merging of "!fixup" or "!squash" commits. A github actions
+    job is employed to block pull requests from being merged if such commits are
+    in the PR branch. Rebase should be used to clean those up.
+
+  It is explicitly _not_ forbidden to have merge-commits in a pull request.
+  Long-running pull requests that contain a large number of changes and are
+  developed over a long period might contain merges. They might even merge
+  `master`/`main` to get up-to-date with the latest developments. Though it is
+  not encouraged to have such long-running pull requests and discouraged to
+  merge `master`/`main` into a PR branch, the project acknowledges that
+  sometimes it is not avoidable.
+
+  To summarize, the following requirements have to be met before a pull request
+  can be merged:
+
+  * Reviews of the relevant persons (ensured via github setting)
+  * Status checks (CI, lints, etc) (ensured via github setting)
+      * builds, tests, lints are green (ensured via github action)
+      * Commit linting (ensured via github action)
+      * No missing `Signed-off-by` lines (ensured via github action)
+      * No "!fixup"/"!squash" commits in the pull request (ensured via github
+        action)
+
+  Merging itself is implemented via a "merge bot": [bors-ng](https://bors.tech).
+  bors-ng is used to prevent "merge skew" or "semantic merge conflicts"
+  (read more [here](https://bors.tech/essay/2017/02/02/pitch/)).
+- **Dependency updates**
+  Dependencies should be kept in sync over all crates in the project. That means
+  that different crates of the project should try to use dependencies in the
+  same versions, but also that dependencies should be harmonized in a way that a
+  specific problem should not be solved with more than one external library at a
+  time.
+  Updates of dependencies is automated via a github bot
+  ([dependabot](https://github.com/dependabot)).
+  To ensure harmonization of dependencies, a dedicated team (see "Team
+  Structure") is responsible for keeping an eye on the list of dependencies.
```

Comment:
```suggestion
  To ensure harmonization of dependencies, a dedicated team for 
  keeping an eye on the list of dependencies can be appointed
  by the maintainers of the project (see "Team Structure").
```

Maybe we should use a more defensive wording here.

Reply:
I feel the current wording is clear enough.